pkg/client/session.go

```diff
@@ -50,8 +62,18 @@ func (c *immuClient) OpenSession(ctx context.Context, user []byte, pass []byte,
 	}
 
 	c.SessionID = resp.GetSessionID()
+	defer func(){
```

Comment:
probably it'd be better to set a local variable and set the attribute in the client when all steps succeeded

Reply:
Yep, there are several approach, I follow @byo suggestion but I can change: Who vote ?

Reply:
I like the current approach with separate variables :+1: